model/scheme.go

```diff
@@ -5,18 +5,22 @@ package model
 
 import (
 	"encoding/json"
+	"fmt"
 	"io"
+	"regexp"
 )
 
 const (
-	SCHEME_NAME_MAX_LENGTH        = 64
-	SCHEME_DESCRIPTION_MAX_LENGTH = 1024
-	SCHEME_SCOPE_TEAM             = "team"
-	SCHEME_SCOPE_CHANNEL          = "channel"
+	SCHEME_DISPLAY_NAME_MAX_LENGTH = 64
+	SCHEME_NAME_MAX_LENGTH         = 64
+	SCHEME_DESCRIPTION_MAX_LENGTH  = 1024
+	SCHEME_SCOPE_TEAM              = "team"
+	SCHEME_SCOPE_CHANNEL           = "channel"
 )
 
 type Scheme struct {
 	Id                      string `json:"id"`
+	DisplayName             string `json:"display_name"`
```

Comment:
Really minor request, but can you reorder DisplayName/Name in this struct to be the other way around so the table columns inSQL are the same order as for other models? Just makes fiddling around in the database a bit easier.